File: Java_Course/src/App/Arrays.java
package App;

import java.util.Scanner;

public class Arrays {
	public static void arrays(String [] args) {
		
		Scanner keyboardInput = new Scanner(System.in);
		
		String [] friends = { "Bruce", "Tony Stark", "Yoda"};
		
		System.out.println(friends[0]);
		System.out.println( friends.length );
		
		friends[0] = "Bruce Wayne";
		System.out.println(friends[0]);
		
		String foes [] = new String[3];
		foes[0] = "King Pin";
		foes[1] = "Joker";
		System.out.println(foes[2]);
		foes[2] = "Thanos";
		
		System.out.println(foes[0]);
		
		System.out.println();
	}
}

File: Ruby_Course/Basics/arrays.rb
friends = Array["Bruce", "Tony Stark", "Peter Parker", "Luke Skywalker"]
puts(friends)

mixed = Array[1, "Karen", true]

puts friends[0]
puts friends[-1]
puts friends[0,2] #stops at 1!!

friends[0] = "Bruce Wayne"
puts friends[0]
friends[3] = "Clark Kent"
puts friends[3]
friends[4] = "Hulk"

foes = Array.new
foes[0] = "Thanos"
foes[1] = "Joker"
foes[5] = "Darth Vader"
puts foes

foes[2] = "King Pin"
foes[3] = "Red Skull"
foes[4] = "Jabba the Hutt"
foes[5] = "Lex Luthor"

puts ("I have " + friends.length().to_s + " allies!")
puts friends.include? "Bruce Wayne"
puts friends.include? "Master Yoda"

puts friends.reverse()
puts foes.sort()

foes[6] = "Winter Soldier"
puts foes
puts "\n"
friends.push(foes.pop())
puts "\nFriends: "
puts friends
puts " \nFoes: "
puts foes
